### PR TITLE
chore(release): bump carbon to v10.15.0

### PR DESCRIPTION
**Pull request by dakahn**
Let's upgrade Carbon's version to 10.15.0 and make sure everything is cool about that 🏄

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,16 +39,16 @@
...
+    "@carbon/pictograms-react": "^10.14.0-rc.0",
     "@loadable/babel-plugin": "^5.12.0",
     "@loadable/component": "^5.12.0",
     "@loadable/webpack-plugin": "^5.12.0",
     "@now/node": "^1.6.1",
     "@philpl/buble": "^0.19.7",
     "babel-preset-gatsby": "^0.2.29",
-    "carbon-components": "^10.14.0",
-    "carbon-components-react": "^7.14.0",
+    "carbon-components": "^10.15.0-rc.0",
```
Comment (joshblack) [inline package.json:50]:
```suggestion
    "carbon-components": "^10.15.0",
```

File: package.json [hunk 2]
```diff
@@ -39,16 +39,16 @@
...
+    "carbon-components-react": "^7.15.0-rc.0",
```
Comment (joshblack) [inline package.json:51]:
```suggestion
    "carbon-components-react": "^7.15.0",
```

File: package.json [hunk 3]
```diff
@@ -39,16 +39,16 @@
     "@carbon/elements": "^10.14.0",
     "@carbon/icons": "^10.13.0",
     "@carbon/icons-react": "^10.13.0",
-    "@carbon/pictograms": "^10.13.0",
-    "@carbon/pictograms-react": "^10.13.0",
+    "@carbon/pictograms": "^10.14.0-rc.0",
+    "@carbon/pictograms-react": "^10.14.0-rc.0",
```
Comment (joshblack) [inline package.json:43]:
```suggestion
    "@carbon/pictograms-react": "^10.14.0",
```

File: package.json [hunk 4]
```diff
@@ -39,16 +39,16 @@
     "@carbon/elements": "^10.14.0",
     "@carbon/icons": "^10.13.0",
     "@carbon/icons-react": "^10.13.0",
-    "@carbon/pictograms": "^10.13.0",
-    "@carbon/pictograms-react": "^10.13.0",
+    "@carbon/pictograms": "^10.14.0-rc.0",
```
Comment (joshblack) [inline package.json:42]:
```suggestion
    "@carbon/pictograms": "^10.14.0",
```

**Comment by vpicone** [comment]
closing this since, updated in a seperate PR

**Pull request closed without merging**